src/editor.ts

```diff
@@ -14,7 +14,7 @@ export class SunSynkCardEditor extends ScopedRegistryHost(LitElement) implements
     @state() private _config?: sunsynkPowerFlowCardConfig;
 
     public setConfig(config: sunsynkPowerFlowCardConfig): void {
-        this._config = config;
+        this._config = {...this._config, ...config};
```

Comment:
Although the emitted change is of type `sunsynkPowerFlowCardConfig`, it first here takes all the values from this original config, then overwrites any properties with the new values from the event.

Reply:
@Pho3niX90  The problem is those values are not defined in the config. I made the following change so that it does not overwrite the full config

https://github.com/slipx06/sunsynk-power-flow-card/blob/1871368e6c3e99e096d6db92a318395a29e34b2f/src/editor.ts#L382-L385